source/bindbc/imgui/dynload.d

```diff
@@ -92,7 +92,7 @@ ImGuiSupport loadImGui()
         ];
     }
     else version(Posix) {
-        const(char)[][2] libNames = [
+        const(char)[][1] libNames = [
```

Comment:
This whole file is actually generated by the generator. This is part of the preamble section and as such should be modified there:

https://github.com/Inochi2D/bindbc-imgui/blob/main/generator/generator.d#L164